exporter/elasticsearchexporter/internal/objmodel/objmodel.go

```diff
@@ -175,6 +176,15 @@ func (doc *Document) AddAttribute(key string, attribute pcommon.Value) {
 	}
 }
 
+// AddEvents converts and adds span events to the document.
+func (doc *Document) AddEvents(key string, events ptrace.SpanEventSlice) {
+	for i := 0; i < events.Len(); i++ {
+		e := events.At(i)
+		doc.Add(flattenKey(key, e.Name()+".time"), TimestampValue(e.Timestamp().AsTime()))
```

Comment:
Why time and not timestamp?

Reply:
currently, any time in span storage with `date time` format in Elasticsearch json documents.  and I think `data time` format is more readable.

Reply:
data time representation is good. My Q is about naming conventions. Do we have any policy on translating the OTel schema to the elastic? So far, I see that proto representation in CamelCase is used. Here it's a bit different, especially for `dropped_attributes_count`.

Do you think it's something that needs to be defined and updated in the exporter? If so, it'd be good to have an issue for that.

Reply:
Looks like we have some rules in the spac https://github.com/open-telemetry/opentelemetry-specification/blob/773ee656f92c7f591f2fd9c38df82c264a15184d/specification/logs/data-model-appendix.md#elastic-common-schema which still need to be completed, but at least we have something to start with

Reply:
thx, I got you, I reviewed https://github.com/open-telemetry/opentelemetry-specification/blob/773ee656f92c7f591f2fd9c38df82c264a15184d/specification/logs/data-model-appendix.md#elastic-common-schema and found that  `dropped_attributes_count` as nested fields is consistent with the style with `cloud.availability_zone`, `error.stack_trace`  in the log model. 

BTW, I agree with we need to have an issue as a good start. But, this issue should be placed in `opentelemetry-specification` repo or `otel-col-contrib`? I guess `opentelemetry-specification`? And It should be completed in another PR?

Reply:
First, we need an issue in contrib to align the schema used in elastic search with what we already have in the spec (or at least as close as possible). Anything that is unclear or missing in the spec should be updated. Fill free to raise issues or PRs there too

Reply:
Yes, I track it with the issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/22137, and I will PR ASAP.

Reply:
Thanks!